articles/specs/launch.0.1.1.xsd

```diff
@@ -8,80 +8,82 @@
     </xs:documentation>
   </xs:annotation>
 
-  <xs:element name="launch" maxOccurs="1">
```

Comment:
Why should this be removed?

Reply:
Because it is not valid XSD. It is already the case as it stands that only one root `launch` element is allowed.